modules/nw-sriov-configuring-operator.adoc

```diff
@@ -59,32 +49,34 @@ Follow this procedure to disable it.
 * You must have installed the SR-IOV Operator.
 
 .Procedure
-. Update the default SriovOperatorConfig Custom Resource
+
+. To update the `default` SriovOperatorConfig CR, enter the following command:
 +
 ----
-$ oc patch sriovoperatorconfig default --patch '{ "spec": { "enableInjector": false}}' --type=merge -n openshift-sriov-network-operator
+$ oc patch sriovoperatorconfig default \
+  --type=merge -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableInjector": false } }'
 ----
 
-. To re-enable Network Resources Injector
+. To reenable the Network Resources Injector, enter the following command:
```

Comment:
I think you had it right the first time. The IBM style guide says to use hyphens when two of the same letters are next to each other.

> Generally, hyphenate a prefix only to prevent misreading or ambiguity. Specifically, hyphenate a prefix in the following situations:
> • In some words in which identical letters would be next to each other (for example, anti-intellectual, bell-like, co-opt, de-emphasize)